docs-kits/kits/DCM-Kit/adoption-view/qna.md

```diff
@@ -0,0 +1,113 @@
+|2|Where can I read about DCM?|<https://catenax-ev.github.io/docs/next/standards/CX-0128-DemandandCapacityManagementDataExchange> contains the released standards. The DCM standard has the ID CX-0128.|
+|3|Are there translations of the DCM standard?|There are no official translations. Only the original standard, written in English, is normative.|
+|4|Is there a standard “easy to read” DCM presentation available?|No, however the standard contains an abstract, which provides a management overview. The KIT itself is receiving regular updates.|
+|5|What is the benefit of using DCM within the Catena-X DataSpace instead of using my individual own process?| Adopting DCM, as defined by Catena-X, has the unique benefit of eliminating the need for defining process and technical interfaces individually with every one of your business partners. This allows you to execute demand and capacity management with every single one of your business partners in an identical manner, greatly reducing overhead.|
+|6|Is there any step-by-step onboarding instruction?|Please have a look at <https://catena-x.net/en/participate>|
+|7|What are the needed steps for onboarding regarding the DCM standard?| Before beginning data exchange, the following conditions must be met: <br/> - Customer and supplier must register with a Catena-X operational company and conform to the Catena-X guidelines <br/> - Customer and supplier must have entered into a contract with each other <br/> - Customer and supplier must sign framework agreements required by Catena-X <br/> - Customer and supplier must have the technical capability to engage in the DCM process with their business applications. You can either enable and certify your IT-system to be Catena-X compliant, or use a certified DCM application. <br/> <br/> You can receive onboarding support by Consultancy Services or Business Application Provider. The operating company has dedicated marketplaces for [certified services](https://www.cofinity-x.com/service-marketplace/) and [certified business applications](https://www.cofinity-x.com/app-marketplace/).|
+|8|Do I need to be the part of Catena-X if I only want to use DCM?|Yes, joining the Catena-X dataspace (not the Catena-X Association) via an operating company is a prerequisite for participating in any Catena-X usecase.|
+|9|Is Cofinity-X required for DCM? Or are there other possibilities?| Currently, Cofinity-X is the only operating company of the Catena-X dataspace and therefore registration at Cofinity-X is a prerequisite for data exchange.|
+|10|Where can I get Catena-X related consultancy services|You can find Catena-X related consultancy services within the Cofinity-X [service marketplace](https://www.cofinity-x.com/service-marketplace/).|
```

Comment:
@jSchuetz88 @stephanbcbauer : Are there also list of certified solution provider / apps, advisory provider in the CX e.V.? I would rather use the neutral website such as the C-X e.V. website? Please advise!

Reply:
As far as I know, we currently do not provide a list of qualified advisory services (I will cross-check). One option, however, would be the following link: https://catena-x.net/en/catena-x-introduce-implement/certification

Reply:
This link could be references: https://catena-x.net/en/catena-x-introduce-implement/certification
The last figure provides a brief overview of the current qualified advisory services

Reply:
> This link could be referenced: https://catena-x.net/en/catena-x-introduce-implement/certification Unfortunately, the overview is not comprehensive, but the last figure provides a general overview.

I do not think, that the suggested certification link works within the context and intend of the quoted Q&A section.
If the association, at some point, keeps a record of consulting companies, I will be more than happy to include that. But for now, the cofinity-x references need to remain.